src/cpp/database/database.cpp

```diff
@@ -5305,6 +5342,7 @@ Info Database::get_info(
     }
 
     return info;
+
 }
```

Comment:
Remove extra line break
```suggestion
    return info;
}
```